### PR TITLE
Creates the ubuntu setup guide

### PR DESCRIPTION
**Pull request by sergiomurilovarela**

### CODE REVIEW

File: ubuntu-setup.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - Android 6.0 (Marshmallow) SDK Platform 23
+  - Google APIs 23
+  - Build-tools 23
+  - ADB
+```
+~/android/tools/bin/sdkmanager "platforms;android-23" "build-tools;23.0.1" "add-ons;addon-google_apis-google-23"
+```
+
+```
+sudo apt-get install android-tools-adb
```
Comment (jhonytaveira) [inline ubuntu-setup.md]: Put genymotion as an option to emulate. 
https://www.genymotion.com/

File: ubuntu-setup.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## Prerequisites:
+#### [Java Development Kit](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+
+#### [Android SDK*](https://developer.android.com/studio/index.html)
+>* Command line tool required only
+
+- extract the downloaded file to `~/android`
+- Add the ANDROID_HOME environment variable:
+```
```
Comment (talyssonoc) [inline ubuntu-setup.md]: Maybe use ```sh?

File: ubuntu-setup.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+~/android/tools/bin/sdkmanager "platforms;android-23" "build-tools;23.0.1" "add-ons;addon-google_apis-google-23"
+```
+
+```
+sudo apt-get install android-tools-adb
+```
+
+## Installation:
+```
+npm install -g react-native-cli
```
Comment (talyssonoc) [inline ubuntu-setup.md]: I think it's worth to say that Node is necessary. Also you'll probably need `sudo` to install global packages.

File: ubuntu-setup.md (hunk 1 above)
Comment (sergiomurilovarela) [inline ubuntu-setup.md]: We can create another file to list options to emulate devices and link it here.

File: ubuntu-setup.md (hunk 3 above)
Comment (felipesoares6) [inline ubuntu-setup.md]: What do you think about list yarn as an option to install de dependencies packages?

File: ubuntu-setup.md (hunk 2 above)
Comment (sergiomurilovarela) [inline ubuntu-setup.md]: Done!

File: ubuntu-setup.md (hunk 3 above)
Comment (sergiomurilovarela) [inline ubuntu-setup.md]: Done!

File: ubuntu-setup.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+> Command line tool required only.
+
+- extract the downloaded file to ```~/android```
+- Add the ANDROID_HOME environment variable:
+```sh
+echo "export ANDROID_HOME=~/android" >> ~/.bashrc &&
+echo "export PATH=$PATH:$ANDROID_HOME/tools" >> ~/.bashrc &&
+echo "export PATH=$PATH:$ANDROID_HOME/platforms"  >> ~/.bashrc
+```
+- Download the SDK necessary packages
```
Comment (willamesoares) [inline ubuntu-setup.md]: What about `Download the required SDK packages` ?

**Pull request merged**